src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rateOfRent = 40;
+  const discountFourthDay = 20;
+  const discountSeventhDay = 50;
+
+  switch (true) {
+    case days < 3 :
+      return days * rateOfRent;
+    case days >= 3 && days < 7 :
+      return days * rateOfRent - discountFourthDay;
+    case days >= 7 :
+      return days * rateOfRent - discountSeventhDay;
+  }
```

Comment:
`switch (true)` isn't a good code, use `if` here, you need only two `ifs` for `days >= 7` and `days >= 3`